### PR TITLE
User table docs

### PR DESCRIPTION
**Pull request by wathen**
#### Description of Work

Adds user table docs

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?

### CODE REVIEW

**Comment by AndrewLister-STFC** [comment]
Looks like this was branched from master, so it has brought all the changes to this branch?
I've just merged the changes from master into 410_... so it's up to date, but there are conflicts? Does this do more than just the docs (can't remember if you told me it did)

**Comment by wathen** [comment]
> Looks like this was branched from master, so it has brought all the changes to this branch?
> I've just merged the changes from master into 410_... so it's up to date, but there are conflicts? Does this do more than just the docs (can't remember if you told me it did)

Yes I think I merged master into 410_...., This PR pulls the rst descriptions from the docs and adds them to the html output

File: docs/source/users/output/acc.rst [hunk 1]
```diff
@@ -4,4 +4,8 @@
 Accuracy Table
 ##############
 
-Place holder for the accuracy table.
+The accuracy results are calculated from the final chi squared value:
+
+.. math:: \min_p \sum_{i=1}^n \left( \frac{y_i - f(x_i, p)}{e_i} \right)^2
+
+where :math:`n` data points :math:`(x_i,y_i)`, associated errors :math:`e_i`, and a model function :math:`f(x,p)`, we find the optimal parameters in the least-squares sense by solving:
```
Comment (AndrewLister-STFC) [inline docs/source/users/output/acc.rst]:
```suggestion
where :math:`n` data points :math:`(x_i,y_i)`, associated errors :math:`e_i`, and a model function :math:`f(x,p)`.
```

File: docs/source/users/output/local_min.rst [hunk 2]
```diff
@@ -4,4 +4,10 @@
 Local Minimiser Table
 #####################
 
-Place holder for the local minimiser table.
+The local min results show a ``True`` or ``False`` values together with a number. The ``True`` or ``False`` indicates whether the software finds a minimum with respect to the following criteria:
```
Comment (AndrewLister-STFC) [inline docs/source/users/output/local_min.rst]:
```suggestion
The local min results shows a ``True`` or ``False`` value together with a number. The ``True`` or ``False`` indicates whether the software finds a minimum with respect to the following criteria:
```

File: docs/source/users/output/local_min.rst [hunk 3]
```diff
@@ -4,4 +4,10 @@
...
+
+    - :math:`||r|| \leq RES\_TOL`,
```
Comment (AndrewLister-STFC) [inline docs/source/users/output/local_min.rst]: These look odd in the math markdown, could RES_TOL and GRAD_TOL be \text{ … } or some other non italic style

File: docs/source/users/output/local_min.rst [hunk 4]
```diff
@@ -4,4 +4,10 @@
...
+    - :math:`|| J^T r|| \leq GRAD\_TOL`,
+    - :math:`\frac{|| J^T r||}{||r||} \leq GRAD\_TOL`,
+
+where :math:`J` and :math:`r` is the Jacobian and residual, respectively. The tolerances can be found in the results object.
```
Comment (AndrewLister-STFC) [inline docs/source/users/output/local_min.rst]:
```suggestion
where :math:`J` and :math:`r` are the Jacobian and residual of :math:`f(x, p)`, respectively. The tolerances can be found in the results object.
```

File: fitbenchmarking/templates/table_descriptions.rst [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+compare: Start
+
+The combined results show the accuracy in the first line of the cell and the runtime on the second line of the cell.
+
+compare: End
+
+local_min: Start
+
+The local min results shows a ``True`` or ``False`` value together with a number. The ``True`` or ``False`` indicates whether the software finds a minimum with respect to the following criteria:
```
Comment (wathen) [inline fitbenchmarking/templates/table_descriptions.rst]:
```suggestion
The local min results shows a ``True`` or ``False`` value together with :math:`\frac{|| J^T r||}{||r||}`. The ``True`` or ``False`` indicates whether the software finds a minimum with respect to the following criteria:

```

File: fitbenchmarking/templates/table_template.html [hunk 6]
```diff
@@ -15,11 +17,33 @@
...
+                            {{ table_format }}
+                        </div>
+                    </section>
+                {% endif %}
                 <center>
                 <div class="table">
                     {{ table }}
                 </div>
-                </center>
+                z  </center>
```
Comment (wathen) [inline fitbenchmarking/templates/table_template.html]:
```suggestion
                </center>
```

File: fitbenchmarking/templates/table_descriptions.rst [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+- :math:`|| J^T r|| \leq \mbox{GRAD\_TOL}`,
+- :math:`\frac{|| J^T r||}{||r||} \leq \mbox{GRAD\_TOL}`,
+
+where :math:`J` and :math:`r` are the Jacobian and residual of :math:`f(x, p)`, respectively. The tolerances can be found in the results object.
+
+local_min: End
+
+runtime: Start
+
+The timing results are calculate from an average using the `timeit <https://docs.python.org/2/library/timeit.html>`_  module in python. The number of runs can be set in :ref:`options`.
```
Comment (wathen) [inline fitbenchmarking/templates/table_descriptions.rst]:
```suggestion
The timing results are calculated from an average using the `timeit <https://docs.python.org/2/library/timeit.html>`_  module in python. The number of runs can be set in :ref:`options`.
```

File: fitbenchmarking/templates/table_template.html [hunk 8]
```diff
@@ -5,6 +5,8 @@
     <link rel="stylesheet" type="text/css" href="{{ css_style_sheet }}" />
     <link rel="stylesheet" type="text/css" href="{{ table_style }}" />
     <link rel="stylesheet" type="text/css" href="{{ custom_style }}" />
+    <link rel="stylesheet" type="text/css" href="{{ maths_style }}" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (wathen) [inline fitbenchmarking/templates/table_template.html]:
```suggestion
    >
```

File: docs/source/users/output/acc.rst [hunk 9]
```diff
@@ -4,4 +4,6 @@
 Accuracy Table
 ##############
 
-Place holder for the accuracy table.
+.. include:: ../../../../fitbenchmarking/HTML_templates/table_descriptions.rst
```
Comment (AndrewLister-STFC) [inline docs/source/users/output/acc.rst]: This is now out of date?

Comment (AndrewLister-STFC) [inline docs/source/users/output/acc.rst]:
```suggestion
.. include:: ../../../../fitbenchmarking/templates/table_descriptions.rst
```

File: docs/source/users/output/compare.rst [hunk 10]
```diff
@@ -4,4 +4,6 @@
 Comparison Table
 ################
 
-Place holder for the compare table.
+.. include:: ../../../../fitbenchmarking/HTML_templates/table_descriptions.rst
```
Comment (AndrewLister-STFC) [inline docs/source/users/output/compare.rst]:
```suggestion
.. include:: ../../../../fitbenchmarking/templates/table_descriptions.rst
```

File: docs/source/users/output/local_min.rst [hunk 11]
```diff
@@ -4,4 +4,7 @@
 Local Minimiser Table
 #####################
 
-Place holder for the local minimiser table.
+.. include:: ../../../../fitbenchmarking/HTML_templates/table_descriptions.rst
```
Comment (AndrewLister-STFC) [inline docs/source/users/output/local_min.rst]:
```suggestion
.. include:: ../../../../fitbenchmarking/templates/table_descriptions.rst
```

File: docs/source/users/output/runtime.rst [hunk 12]
```diff
@@ -4,4 +4,6 @@
 Runtime Table
 #############
 
-Place holder for the runtime table.
+.. include:: ../../../../fitbenchmarking/HTML_templates/table_descriptions.rst
```
Comment (AndrewLister-STFC) [inline docs/source/users/output/runtime.rst]:
```suggestion
.. include:: ../../../../fitbenchmarking/templates/table_descriptions.rst
```

**Pull request merged**